ovirt-ansible-disaster-recovery.spec.in

```diff
@@ -26,10 +26,12 @@ This Ansible role provide funtionality to perform disaster recovery for oVirt en
 export PKG_DATA_DIR_ORIG=%{_datadir}/%{ansible_roles_dir}/%{roleprefix}%{rolename}
 export PKG_DATA_DIR=%{buildroot}$PKG_DATA_DIR_ORIG
 export PKG_DOC_DIR=%{buildroot}%{_pkgdocdir}
+install -dm 755 "$PKG_LOG_DIR/$LOG_NAME/$LOG_NAME.log"
```

Comment:
Please add `Requires: logrotate` spec file

Reply:
done

Reply:
You need to use `Requires: logrotate`, logrotate is a package name